ykman-gui/qml/SnackBarError.qml

```diff
@@ -43,6 +43,8 @@ SnackBar {
             return qsTr('Wrong PIN. Tries remaning: %1'.arg(resp.tries_left))
         case 'wrong_puk':
             return qsTr("Wrong PUK. Tries remaning: %1".arg(resp.tries_left))
+        case 'failed_parsing':
+            return qsTr("Something went wrong with reading the file")
```

Comment:
Maybe rewrite this as something like "Failed to parse the file"?

Reply:
Done!